### PR TITLE
Add encoder configurations

### PR DESCRIPTION
**Pull request by nasirhemed**
# Add different encoder configurations

* Added configurations for svt-rt and rav1e-rt in `encoder_commands`
* Updated a way to get associated encoder_command by using a function instead of dictionary 
* Add binary constants for svt-rt and rav1e

Since rav1e only accepts y4m videos as inputs, add a process to always
convert yuv to y4m and pass that as input to rav1e.

### CODE REVIEW

File: rtc-video-quality/setup_rav1e.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+# Copyright 2017 Google Inc. All rights reserved.
```
[the remark this answers is not in the document]
Comment (nasirhemed) [reply rtc-video-quality/setup_rav1e.sh:2]: Yeah sure will do for SVT and rav1e setup scripts

File: rtc-video-quality/encoder_commands.py [hunk 2]
```diff
@@ -0,0 +1,433 @@
...
+      '-o', encoded_filename,
+      '-b', job['target_bitrates_kbps'][0],
+    ]
+    encoded_files = [{'spatial-layer': 0, 'temporal-layer': 0, 'filename': encoded_filename}]
+    return ([str(i) for i in command], encoded_files)
+
+def get_encoder_command(encoder):
+    encoders = [
+        'aom-good', 'aom-rt', 'aom-all_intra', 'aom-offline', ## AOM CONFIGS
+        'rav1e-1pass', 'rav1e-rt', 'rav1e-all_intra', ## RAV1E CONFIGS TODO: FIXME
```
[the remark this answers is not in the document]
Comment (nasirhemed) [reply rtc-video-quality/encoder_commands.py:412]: Noted. This todo has already been done. but I'll remove any TODO's that are done and update the changes

File: rtc-video-quality/generate_data.py [hunk 3]
```diff
@@ -392,256 +260,309 @@ def add_framestats(results_dict, framestats_file, statstype):
...
+    temporal_skip = temporal_divide - 1
+    # TODO(pbos): Perform SSIM on downscaled .yuv files for spatial layers.
+    (fd, metrics_framestats) = tempfile.mkstemp(dir=temp_dir, suffix=".csv")
+    os.close(fd)
+    ssim_results = subprocess.check_output([
+        TINY_SSIM_BIN, clip['yuv_file'], decoded_file,
+        "%dx%d" % (results_dict['width'], results_dict['height']),
+        str(temporal_skip), metrics_framestats
+    ],
+                                           encoding='utf-8').splitlines()
```
[the remark this answers is not in the document]
Comment (nasirhemed) [reply rtc-video-quality/generate_data.py:277]: I'm not sure why that happened. I'm currently using yapf to format files

[the remark this answers is not in the document]
Comment (todd-n) [reply rtc-video-quality/generate_data.py:277]: That should be fine then, maybe add a blank line after line 277 to make it easier to read. Line 314 too

File: rtc-video-quality/setup_svt-av1.sh [hunk 4]
```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+# Copyright 2017 Google Inc. All rights reserved.
```
[the remark this answers is not in the document]
Comment (nasirhemed) [reply rtc-video-quality/setup_svt-av1.sh:2]: Change made in previous [pull request](https://github.com/googleinterns/av1-codec-comparison/pull/4/commits/9a2699b2f4b6d0fc6962c8f9da9a066993a16e0f)

File: rtc-video-quality/generate_data.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (nasirhemed) [reply rtc-video-quality/generate_data.py:277]: Change made to previous [pull request](https://github.com/googleinterns/av1-codec-comparison/pull/4/commits/b609b75118dbfdc332aa2c8fad785dee2f07a836)

**Comment by nasirhemed** [comment]
I'll close this pull request and reopen it once the other pull requests are done. It's complicated to manage the commits from different pull request as it results in a lot of conflicts.

I'll reopen it once the other pull requests are merged

File: rtc-video-quality/setup_rav1e.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (nasirhemed) [reply rtc-video-quality/setup_rav1e.sh:2]: Change made in previous [pull request](https://github.com/googleinterns/av1-codec-comparison/pull/4/commits/9a2699b2f4b6d0fc6962c8f9da9a066993a16e0f)

**Pull request merged**